### PR TITLE
Map Circle Warning 

### PR DESCRIPTION
**Pull request by JGibbsWork**
Updated signup to include a confirmation dialogue wrapper warning about the circle surrounding the address. Not sure if we want to create a barrier for signing up, but it did seem like a more direct way to solve the problem for new users. Also added text below the map when editing your profile. More than happy to change either accordingly.

**Frontend checklist**
- [x] Formatted my code with `yarn format && yarn lint --fix`
- [x] There are no warnings from `yarn lint`
- [ ] There are no console warnings when running the app
- [ ] Added any new components to storybook
- [ ] Added tests where relevant
- [ ] All tests pass
- [ ] Clicked around my changes running locally and it works
- [ ] Checked Desktop, Mobile and Tablet screen sizes

### CODE REVIEW

**Comment by lucaslcode** [comment]
@AllhandsNoFeet thoughts?

**Review by lucaslcode — CHANGES_REQUESTED** [review]
Seems good to me for now.

Could you change `ConfirmationDialogWrapper.tsx` so it has a prop that lets you choose between "Okay" and "Yes" as the 'accept button'. Like `confirmButtonLabel`

I also noticed that if there's a server error ("you must be 18 yo to sign up"), it is hard to notice because the submit button is at the bottom of a long form.

Could you add this to `signup.tsx`

```
  useEffect(() => {
    if (authState.error) window.scroll({ top: 0, behavior: "smooth" });
  }, [authState.error]);
```

File: app/frontend/src/components/EditLocationMap.tsx [hunk 1]
```diff
@@ -363,6 +364,9 @@ function RadiusSlider({ commit, initialRadius, redrawMap }: RadiusSliderProps) {
   const [radius, setRadius] = useState(initialRadius);
   return (
     <>
+      <Typography color="secondary" gutterBottom={true}>
```
Comment (lucaslcode) [inline app/frontend/src/components/EditLocationMap.tsx]:
```suggestion
      <Typography variant="body2" gutterBottom>
```
I think that's sufficient

**Comment by JGibbsWork** [comment]
Cool on it!

File: app/frontend/src/components/ConfirmationDialogWrapper.tsx [hunk 2]
```diff
@@ -37,7 +39,9 @@ export default function ConfirmationDialogWrapper({
           <DialogContentText>{message}</DialogContentText>
         </DialogContent>
         <DialogActions>
-          <Button onClick={handleConfirm}>Yes</Button>
+          <Button onClick={handleConfirm}>
+            {confirmButtonLabel ? confirmButtonLabel : "Yes"}
```
Comment (lucaslcode) [inline app/frontend/src/components/ConfirmationDialogWrapper.tsx]: Better move the default to a constant (sorry)

**Review by lucaslcode — CHANGES_REQUESTED** [review]
Tiny thing but looks good!

**Review by lucaslcode — APPROVED** [review]
👍

**Comment by aapeliv** [comment]
Is this ready to merge? If so, feel free to merge it yourself @JGibbsWork

**Comment by JGibbsWork** [comment]
Sorry about that! All set now!

**Pull request merged**